daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -475,6 +475,27 @@ class RazerBlackWidowXChroma(_RippleKeyboard):
 
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/716/716_blackwidow_x_chroma.png"
 
+class RazerHuntsmanV2Tenkeyless(_RippleKeyboard):
+    """
+    Class for the Razer Huntsman V2 Tenkeyless
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Huntsman_V2_Tenkeyless(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x026B
+    HAS_MATRIX = True
+    DEDICATED_MACRO_KEYS = False
+    MATRIX_DIMS = [6, 22]
```

Comment:
I am not sure on the MATRIX_DIMS here, the other tenkeyless keyboard is set to 6x18, not 6x22

Reply:
Good point i missed that when adjusting for the tkl